docs/src/tutorials/applications/optimal_power_flow.jl

```diff
@@ -0,0 +1,545 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Optimal power flow
+
+# *This tutorial was originally contributed by James Foster (@jd-foster).*
+
+# This tutorial formulates and solves an alternating current optimal power flow
+# (AC-OPF) problem, a much-studied nonlinear problem from the field of
+# electrical engineering.
+
+# Once we've formulated and solved the nonlinear problem, we will turn our focus
+# to obtaining a good estimate of the objective value at the global optimum
+# through the use of semidefinite programming.
+
+# The purpose of this tutorial is to highlight JuMP's ability to directly
+# formulate problems involving complex-valued decision variables and complex
+# matrix cones such as the  [`HermitianPSDCone`](@ref) object.
+
+# For another example of modeling with complex decision variables, see the
+# [Quantum state discrimination](@ref) tutorial, and see the
+# [Complex number support](@ref) section of the manual for more details.
```

Comment:
You could also link to PowerModels here to make it clear that people don't have to explicitly model using JuMP